ext/web/06_streams.js

```diff
@@ -1708,6 +1837,18 @@
     }
   }
 
+  /**
+   * @template R
+   * @param {ReadableStreamDefaultReader<R>} reader
+   */
+  function readableStreamDefaultReaderRelease(reader) {
+    readableStreamReaderGenericRelease(reader);
+    const e = new TypeError(
+      "There are pending read requests, so the reader cannot be released.",
```

Comment:
This error message is no longer accurate. The reader can (and will) be released, but any pending read requests will be *rejected* with this `TypeError`.

Same applies to `readableStreamBYOBReaderRelease`.

Reply:
Good point, i should have put more thought into that. thanks for pointing it out, will fix ASAP